parquet-hadoop/src/main/java/org/apache/parquet/filter2/statisticslevel/StatisticsFilter.java

```diff
@@ -68,8 +68,8 @@
   private static final boolean BLOCK_CANNOT_MATCH = true;
 
   public static boolean canDrop(FilterPredicate pred, List<ColumnChunkMetaData> columns) {
-    checkNotNull(pred, "pred");
-    checkNotNull(columns, "columns");
+    checkNotNull(pred, "predicate cannot be null");
+    checkNotNull(columns, "columns cannot be null");
```

Comment:
i just realized this isn't even guava preconditions, this is your own checkNotNull. This is definitely the wrong thing to do, will be reverted